#Header header

int64 id 	#ROOmba's id
float64 x	#Roomba's x
float64 y   #Roomba's y

float64 v_x #Roomba's velocity in the x
float64 v_y #Roomba's velocity in the y

bool checked	#Boolean for if the roomba has been checked
bool active		#Boolean for if the roomba is active